src/umd/shims/ReactAddonsDOMDependenciesUMDShim.js

```diff
@@ -9,24 +9,22 @@
  * @providesModule ReactAddonsDOMDependenciesUMDShim
  */
 
-/* globals ReactDOM */
-
 'use strict';
 
-exports.getReactDOM = function() {
-  return ReactDOM;
-};
+function getReactDOM() {
+  var ReactWithAddonsUMDEntry = require('ReactWithAddonsUMDEntry');
```

Comment:
Probably not a huge deal in the grand scheme, but could potentially memoize this so we don't have to enter `require` on every call.